app/App.java

```diff
@@ -3,75 +3,17 @@
 import java.util.Scanner;
 
 import icebase.icebase.User;
+import icebase.app.screens.SplashScreen;
 import icebase.icebase.Auth;
 
 public class App {
-    public static void main(String[] args) {
-        Scanner sc = new Scanner(System.in);
-
-        // Splash Screen
-        MenuTitle.displayMainTitle();
-        System.out.printf("\n\t[1] %s%15s[2] %s\n\n",
-                "Log-in", " ", "Sign-up");
-        System.out.print("Choice: ");
-        sc.nextLine();
-
-        // Pwede nyo i play around ang this shit
-        try {
-            Auth auth = Auth.getAuth();
-
-            // setup sa account
-            // might error if i run mo twice, so just delete lang sa users nga data folder
-            User user = auth.signUp("hi", "hello");
-            user.setStoreId("shitStoreID"); // generated random uid ni siya pero yeah
-            user.setMoney(9999);
-
-            // Actual logging in
-            auth.login("hi", "hello");
-
-            // Pwede na magamit anywhere sa app if need man niya ang signed in user
-            user = auth.getUser();
+    public static Scanner sc = new Scanner(System.in);
```

Comment:
woah static scanner, dapat bang iapply ang this type of implementation to other classes?

Reply:
if makita nimo sa screens kay ang scanner is

`Scanner sc = App.sc`